src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -649,10 +649,23 @@ bool OIIO_API resize (ImageBuf &dst, const ImageBuf &src, Filter2D *filter,
 /// needed, rather than attempting to interpolate deep pixels (regardless of
 /// the value of `interpolate`).
 ImageBuf OIIO_API resample (const ImageBuf &src, bool interpolate = true,
-                        ROI roi={}, int nthreads=0);
+                            ROI roi={},
+                            ImageBuf::WrapMode wrap=ImageBuf::WrapDefault,
```

Comment:
Can you please swap the order here (and everywhere), so that roi and nthreads are always the last two parameters, for uniformity with absolutely everything else?